hardware/blink1/blink1.js

```diff
@@ -4,7 +4,7 @@ module.exports = function(RED) {
     var Blink1 = require("node-blink1");
     var blink1 = {};
 
-    function Blink1Node(n) {
+    function Blink1OutNode(n) {
```

Comment:
can't change name of existing node.

Reply:
It didn't break functionality for me? Existing nodes continued functioning even after changing the function name. Or is there another reason you say I can't change the name of existing node?

Again: Opinionated. I don't like the old name, but it doesn't matter functionality wise.

Reply:
if it isn't broken - don't change it. It is unnecessary for the new functionality.